### ISSUE TITLE
Documentation pointer

### ISSUE THREAD

**Issue by dgieedelman** [rdeits/PyLCM.jl#18]
For a newbie could you point to documentation on how to (on linux in my case)
make sure you really have exlcm
and can install the exlcm package  in your PYTHONPATH
so that @pyimport exlcm
doesn't error out with
ImportError('No module named exlcm',)

**Comment by rdeits** [comment]
Yeah, this would be good to have in the documentation. Most of the information can be found in the original LCM docs: https://lcm-proj.github.io/tut_python.html Does that have what you need?

**Comment by dgieedelman** [comment]
i was able to get the files
 exlcm/example_t.py
  exlcm/__init__.py

but i don't undertand what it means to install the exlcm package in my PYTHONPATH

forgive the newbie question but typing  
export PYTHONPATH=/home/ubuntu/Downloads/lcm-1.3.1/exlcm

didn't quite work ... is that the right exlcm ???

**Comment by rdeits** [comment]
That's close, but you need PYTHONPATH to point to where the Python files produced by lcm-gen are. Specifically, if lcm-gen has created the file `/foo/bar/exlcm/__init__.py` then you want `export PYTHONPATH="$PYTHONPATH:/foo/bar"`

**Comment by dgieedelman** [comment]
thanks for the help!

can i also do this with a pyimport or @pyimport from within julia or must i do it from the shell?

i have a similar question for
/home/ubuntu/.julia/v0.5/PyLCM/test/pylcm_test_message

i tried a few combinations like
unshift!(PyVector(pyimport("sys")["path"]), "/home/ubuntu/.julia/v0.5/PyLCM/test")
unshift!(PyVector(pyimport("sys")["path"]), "pylcm_test_message")
unshift!(PyVector(pyimport("sys")["/home/ubuntu/.julia/v0.5/PyLCM/test"]), "pylcm_test_message")

hoping to import <path> as <name> from within julia, and again i think i'm close but
i'm missing one little bit i think ....

**Comment by rdeits** [comment]
Ok, this isn't really an LCM issue. Your first attempt looks correct, but I don't know if `"/home/ubuntu/.julia/v0.5/PyLCM/test"` is the right path. 

My suggestion would be to first make sure you can:

* create a `.py` file
* add its location to the PYTHONPATH using `unshift!(PyVector...`
* import it with `@pyimport`

and then making this work for LCM types will be easy.
